frontend/src/styles/layout/_shell.scss

```
@@ -0,0 +1,93 @@
+.main-content {
+  min-width: 0;
+}
+
+@include respond-down('mobile') {
+  .hamburger {
+    display: block;
+  }
+
+  .main-content {
+    margin-inline-start: 0 !important;
+    // Tighter side gutters on mobile; leave room for the fixed hamburger.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)